opennms-base-assembly/src/main/filtered/bin/send-event.pl

```diff
@@ -35,7 +35,7 @@
 $ZONE    = 'GMT';
 
 @SEVERITIES = ( undef, 'Indeterminate', 'Cleared', 'Normal', 'Warning', 'Minor', 'Major', 'Critical' );
-	
```

Comment:
Can we update `contrib/send-event.py` as well?

Reply:
Sure.  Didn't realize there were 2 send-events.  :)